packages/nocodb/src/db/BaseModelSqlv2.ts

```diff
@@ -4512,6 +4513,20 @@ class BaseModelSqlv2 {
     return true;
   }
 
+  /*
+   *  Utility method to validate database constraints
+   */
+  protected validateConstraints(
+    column: Column<any>,
+    data: Record<string, any>,
+  ) {
+    if (column.clen < data[column.title]?.length) {
```

Comment:
use `dtxp` here - `column.clen` => `+column.dtxp`

Reply:
And do we have to do this for all types? in some cases, this value won't be defined

Reply:
If the value is not defined the expression will evaluate to false right ?

Reply:
Added an explicit check to make it clear